src/test/java/com/jakewharton/rxrelay3/SerializedRelayTest.java

```diff
@@ -169,8 +170,10 @@ public void run() {
 
             TestHelper.race(r1, r2, Schedulers.single());
 
-            ts.assertSubscribed().assertNoErrors().assertNotComplete()
-            .assertValueSet(Arrays.asList(1, 2));
+            List<Integer> actual = ts.assertNoErrors().assertNotComplete().values();
+            List<Integer> expected = Arrays.asList(1, 2);
+            assertTrue("The collections are not the same", actual.size() == expected.size()
+                    && actual.containsAll(expected) && expected.containsAll(actual));
```

Comment:
Why doesn't this use normal equals?

Reply:
This is because the code didn't ensure the position and I didn't want to change the behaviour. But now that I check it with `actual.size() == expected.size() && actual.containsAll(expected)` is enough.